meegpowreg/power_features.py

```diff
@@ -67,8 +67,7 @@ def compute_features(
         fbands={'alpha': (8.0, 12.0)},
         clean_func=lambda x: x,
         n_jobs=1):
-    #doc str
-
+    """Compute features from raw data or clean epochs."""
```

Comment:
you can do better. See an example in 

https://github.com/mne-tools/mne-python/blob/main/mne/event.py#L24

you could also read this page